kyuubi-server/pom.xml

```diff
@@ -486,6 +503,27 @@
             <artifactId>flexmark-all</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>kafka_${scala.binary.version}</artifactId>
```

Comment:
I think we are using Testcontainers for integration testing, why should we depend on the Kafka server?

And after second thought, I prefer to use `MockProducer` rather than launch a real Kafka broker, but the latter is also fine for me.

Reply:
`MockProducer` is lightweight to simulate `producer-consumer` situation, but we need a broker bootstrap server from `producer-borker-consumer` here. And I don't know how to get a bootstrap server from a `MockProducer`.

Reply:
the purpose of the consumer is to verify the events have been successfully delivered to Kafka, we can simply check the buffer of MockProducer instead